glue_vispy_viewers/isosurface/layer_style_widget.py

```diff
@@ -21,6 +21,8 @@ class IsosurfaceLayerStyleWidget(QtGui.QWidget):
     # GUI elements
     attribute = CurrentComboProperty('ui.combo_attribute')
     level = FloatLineProperty('ui.value_level')
+    # We don't have IntLineProperty?
+    step_num = FloatLineProperty('ui.step_num')
```

Comment:
The UI crashed after I added this one line of code, in order to have an input textbox for setting the isosurface layer numbers. @astrofrog Do you know the problem here? It seems the ui element is not found as shown below:

> Traceback (most recent call last):
  File "/Users/penny/Works/Gluedev/glue/glue/glue/core/qt/layer_artist_model.py", line 207, in rowsInserted
    parent.on_artist_add(self.model().artists)
  File "/Users/penny/Works/Gluedev/glue/glue/glue/core/qt/layer_artist_model.py", line 313, in on_artist_add
    self.layout_style_widgets[layer_artist] = self.layer_style_widget_cls(layer_artist)
  File "/Users/penny/anaconda/lib/python2.7/site-packages/glue_vispy_viewers-0.5.dev0-py2.7.egg/glue_vispy_viewers/isosurface/layer_style_widget.py", line 33, in __init__
    directory=os.path.dirname(__file__))
  File "/Users/penny/Works/Gluedev/glue/glue/glue/utils/qt/helpers.py", line 118, in load_ui
    return load_ui(full_path, parent, custom_widgets=CUSTOM_QWIDGETS)
  File "/Users/penny/Works/Gluedev/glue/glue/glue/external/qt.py", line 281, in load_ui
    return _load_ui_pyqt4(path, parent)
  File "/Users/penny/Works/Gluedev/glue/glue/glue/external/qt.py", line 293, in _load_ui_pyqt4
    return loadUi(path, parent)
  File "/Users/penny/anaconda/lib/python2.7/site-packages/PyQt4/uic/__init__.py", line 236, in loadUi
    return DynamicUILoader(package).loadUi(uifile, baseinstance, resource_suffix)
  File "/Users/penny/anaconda/lib/python2.7/site-packages/PyQt4/uic/Loader/loader.py", line 71, in loadUi
    return self.parse(filename, resource_suffix, basedir)
  File "/Users/penny/anaconda/lib/python2.7/site-packages/PyQt4/uic/uiparser.py", line 982, in parse
    actor(elem)
  File "/Users/penny/anaconda/lib/python2.7/site-packages/PyQt4/uic/uiparser.py", line 803, in createUserInterface
    self.traverseWidgetTree(elem)
  File "/Users/penny/anaconda/lib/python2.7/site-packages/PyQt4/uic/uiparser.py", line 781, in traverseWidgetTree
    handler(self, child)
  File "/Users/penny/anaconda/lib/python2.7/site-packages/PyQt4/uic/uiparser.py", line 478, in createLayout
    self.traverseWidgetTree(elem)
  File "/Users/penny/anaconda/lib/python2.7/site-packages/PyQt4/uic/uiparser.py", line 781, in traverseWidgetTree
    handler(self, child)
  File "/Users/penny/anaconda/lib/python2.7/site-packages/PyQt4/uic/uiparser.py", line 521, in handleItem
    self.traverseWidgetTree(elem)
  File "/Users/penny/anaconda/lib/python2.7/site-packages/PyQt4/uic/uiparser.py", line 781, in traverseWidgetTree
    handler(self, child)
  File "/Users/penny/anaconda/lib/python2.7/site-packages/PyQt4/uic/uiparser.py", line 478, in createLayout
    self.traverseWidgetTree(elem)
  File "/Users/penny/anaconda/lib/python2.7/site-packages/PyQt4/uic/uiparser.py", line 781, in traverseWidgetTree
    handler(self, child)
  File "/Users/penny/anaconda/lib/python2.7/site-packages/PyQt4/uic/uiparser.py", line 521, in handleItem
    self.traverseWidgetTree(elem)
  File "/Users/penny/anaconda/lib/python2.7/site-packages/PyQt4/uic/uiparser.py", line 781, in traverseWidgetTree
    handler(self, child)
  File "/Users/penny/anaconda/lib/python2.7/site-packages/PyQt4/uic/uiparser.py", line 256, in createWidget
    self.stack.push(self.setupObject(widget_class, parent, elem))
  File "/Users/penny/anaconda/lib/python2.7/site-packages/PyQt4/uic/uiparser.py", line 226, in setupObject
    setattr(self.toplevelWidget, name, obj)
  File "/Users/penny/Works/Gluedev/glue/glue/glue/utils/qt/widget_properties.py", line 72, in __set__
    widget = reduce(getattr, [instance] + self._att)
AttributeError: 'IsosurfaceLayerStyleWidget' object has no attribute 'ui'